website/src/components/pageStateSelectors/WasapPageStateSelector.tsx

```diff
@@ -271,23 +291,71 @@ function ResistanceMutationsFilter({
 function UntrackedFilter({
     pageState,
     setPageState,
+    cladeLineageQueryResult: { isPending, isError, error, data: cladeLineages },
 }: {
     pageState: WasapUntrackedFilter;
     setPageState: (newState: WasapUntrackedFilter) => void;
+    cladeLineageQueryResult: UseQueryResult<Record<string, string>>;
 }) {
+    const defaultLineages = cladeLineages ? Object.values(cladeLineages) : [];
+
     return (
         <>
             <SequenceTypeSelector
                 value={pageState.sequenceType}
                 onChange={(sequenceType) => setPageState({ ...pageState, sequenceType })}
             />
             <LabeledField label='Known variants to exclude'>
-                <input
-                    className='input input-bordered'
-                    value={pageState.excludeVariants?.join(' ')}
-                    onChange={(e) => setPageState({ ...pageState, excludeVariants: e.target.value.split(' ') })}
-                />
+                <select
+                    className='select select-bordered'
+                    value={pageState.excludeSet}
+                    onChange={(e) => setPageState({ ...pageState, excludeSet: e.target.value as ExcludeSetName })}
+                >
+                    <option value='nextstrain'>Nextstrain clades</option>
+                    <option value='custom'>custom</option>
+                </select>
```

Comment:
Why don't we use the `RadioSelect` select here? This also only has two options. Or do we expect more options in the future?

Reply:
Ok, then I think I'd leave it the way it is for now.